### PR TITLE
ZDI assigns the following CVEs:

### PR DESCRIPTION
**Pull request by zdi-team**
A  CVE-2021-34865.json
A  CVE-2021-34866.json
A  CVE-2021-34867.json
A  CVE-2021-34868.json
A  CVE-2021-34869.json
A  CVE-2021-34870.json

### CODE REVIEW

File: CVE-2021-34865.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (cve-team) [inline CVE-2021-34865.json]: Expected location (2021/34xxx/CVE-2021-34865.json) of file does not match actual location (CVE-2021-34865.json)

File: CVE-2021-34866.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (cve-team) [inline CVE-2021-34866.json]: Expected location (2021/34xxx/CVE-2021-34866.json) of file does not match actual location (CVE-2021-34866.json)

File: CVE-2021-34867.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (cve-team) [inline CVE-2021-34867.json]: Expected location (2021/34xxx/CVE-2021-34867.json) of file does not match actual location (CVE-2021-34867.json)

File: CVE-2021-34868.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (cve-team) [inline CVE-2021-34868.json]: Expected location (2021/34xxx/CVE-2021-34868.json) of file does not match actual location (CVE-2021-34868.json)

File: CVE-2021-34869.json [hunk 5]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (cve-team) [inline CVE-2021-34869.json]: Expected location (2021/34xxx/CVE-2021-34869.json) of file does not match actual location (CVE-2021-34869.json)

File: CVE-2021-34870.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (cve-team) [inline CVE-2021-34870.json]: Expected location (2021/34xxx/CVE-2021-34870.json) of file does not match actual location (CVE-2021-34870.json)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
@zdi-team ,

Please review and correct the issues listed from the Auto validation checker. Once corrected we can proceed with the process of merging these changes into the main list. 

Thanks,
CVE Content Team

File: CVE-2021-34866.json (hunk 2 above)
Comment (zdi-team) [reply CVE-2021-34866.json]: moved to proper location and squashed commit

File: CVE-2021-34867.json (hunk 3 above)
Comment (zdi-team) [reply CVE-2021-34867.json]: moved to proper location and squashed commit

File: CVE-2021-34869.json (hunk 5 above)
Comment (zdi-team) [reply CVE-2021-34869.json]: moved to proper location and squashed commit

File: CVE-2021-34868.json (hunk 4 above)
Comment (zdi-team) [reply CVE-2021-34868.json]: moved to proper location and squashed commit

File: CVE-2021-34870.json (hunk 6 above)
Comment (zdi-team) [reply CVE-2021-34870.json]: moved to proper location and squashed commit

**Comment by zdi-team** [comment]
> @zdi-team ,
> 
> Please review and correct the issues listed from the Auto validation checker. Once corrected we can proceed with the process of merging these changes into the main list.
> 
> Thanks, CVE Content Team



> @zdi-team ,
> 
> Please review and correct the issues listed from the Auto validation checker. Once corrected we can proceed with the process of merging these changes into the main list.
> 
> Thanks, CVE Content Team

Done

**Pull request merged**